stop_and_wait_plugin/src/stop_and_wait_plugin.cpp

```diff
@@ -375,9 +375,9 @@ std::vector<carma_planning_msgs::msg::TrajectoryPlanPoint> StopandWait::compose_
   {
     if (times[i] != 0 && !std::isnormal(times[i]) && i != 0)
     {  // If the time
-      RCLCPP_WARN_STREAM(rclcpp::get_logger("stop_and_wait_plugin"),"Detected non-normal (nan, inf, etc.) time. Making it same as before: " << times[i-1]);
-      // NOTE: overriding the timestamps in assumption that pure_pursuit_wrapper will detect it as stopping case
-      times[i] = times[i - 1];
+      RCLCPP_WARN_STREAM(rclcpp::get_logger("stop_and_wait_plugin"),"Detected non-normal (nan, inf, etc.) time."
+        "This happens due to  dv = dx/0.0 = inf speed. Incrementing time by arbitrary large number to achieve stopping behavior");
+      times[i] = times[i - 1] + 3600.0; //assigning 10hr difference between what is normally 0.1-0.2 sec
```

Comment:
Why are we unable to set the time to `inf`?

Reply:
I'm not sure this correction should be here. I think it should move to the `trajectory_from_points_times_orientations` function. That function generates a trajectory, so it should be the one to take corrective actions if the inputs are collectively invalid.

Reply:
So I inserted this function into the function you specified. I made the `times` value pass by reference because we are modifying it. I know we don't technically use it after the function, so we don't need to pass it by reference. However, I find the log statement that has `downtrack, t, and v` really helpful:
`RCLCPP_DEBUG_STREAM(rclcpp::get_logger("stop_and_wait_plugin"),"1d: " << downtracks[i] << " t: " << times[i] << " v: " << filtered_speeds[i]);`

Reply:
I also ended up adding more logic since you last reviewed that was throwing as well. So I fixed it.

Reply:
1. `speed_to_time` is still setting it to `nan` that part is not changed.
2. you are right, I changed it so that it is incrementing by config_.stop_timestep now
3. I also applied this change and reverted the pass-by-reference change